src/services/helpers.ts

```diff
@@ -182,27 +180,26 @@ export const calculateScore = (vocabularyItemsWithResults: VocabularyItemResult[
   )
 }
 
-const normalizeSearchString = (searchString: string): string => {
-  const searchStringWithoutArticle = ARTICLES.map(article => article.value).includes(
-    searchString.split(' ')[0].toLowerCase()
-  )
-    ? searchString.substring(searchString.indexOf(' ') + 1)
-    : searchString
-  return normalizeStrings(searchStringWithoutArticle).toLowerCase().trim()
-}
-
 export const matchAlternative = (vocabularyItem: VocabularyItem, searchString: string): boolean =>
   vocabularyItem.alternatives.filter(alternative =>
-    alternative.word.toLowerCase().includes(normalizeSearchString(searchString))
+    alternative.word.toLowerCase().includes(
+      searchString
+        .substring(searchString.indexOf(' ') + 1)
+        .toLowerCase()
+        .trim()
+    )
   ).length > 0
 
 export const getSortedAndFilteredVocabularyItems = (
   vocabularyItems: VocabularyItem[] | null,
   searchString: string
 ): VocabularyItem[] => {
-  const collator = new Intl.Collator('de-De', { sensitivity: 'base', usage: 'sort' })
+  const collator = new Intl.Collator('de-De', { sensitivity: 'base', usage: 'sort', localeMatcher: 'lookup' })
 
-  const normalizedSearchString = normalizeSearchString(searchString)
+  const normalizedSearchString = searchString
```

Comment:
By changing this like that, we will have problems: e.g. when entering 
"Die Ölk" the result should be "Die Ölkanne", 
when entering "große Ölkanne", the result should be empty.

In the previous `normalizeSearchString` function this was working properly, as we there only removed the first word from the searchString if it is an Article, see:
```
  const searchStringWithoutArticle = ARTICLES.map(article => article.value).includes(
    searchString.split(' ')[0].toLowerCase()
  )
    ? searchString.substring(searchString.indexOf(' ') + 1)
    : searchString
```
If the first word is an article we remove it from the search string, if the first word is something else (not an article) we keep it in the search string.

This behaviour should not be changes by this PR.